src/domains/checkout-common/env/weu-uat/terraform.tfvars

```diff
@@ -40,3 +40,13 @@ pagopa_proxy_redis_family   = "C"
 redis_private_endpoint_enabled = true
 redis_zones                    = []
 
+# Checkout Redis
+redis_checkout_params = {
+  capacity   = 0
+  sku_name   = "Basic"
+  family     = "C"
+  version    = 6
```

Comment:
Have you considered using Redis 7?
https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-how-to-upgrade

Reply:
It seems that version 7 is available only for Enterprise tier and not for Basic/Standard/Premium ones.
Moreover actually used module only accept 4 and 6 as version, so maybe we should upgrade module first too
Should we use Enterprise version?